gulpfile.babel.js

```diff
@@ -61,9 +74,9 @@ function sass() {
 
 // Copy and compress images
 function images() {
-  return gulp.src('src/assets/img/*')
+  return gulp.src('src/assets/images/*')
```

Comment:
Good call. Originally it was to align to what I thought was needed in the zip task (vendor requirement) however, I now know the images don't even need to be in a folder and as such, if they are the only requirement is that the html refs relatively. 

So I'll revert the dir name change back.  Anything else? Should I be PRing the master branch? Are we at the point were I should include read me updates?